changelog/fix-4629-UPE-methods-selection

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix UPE checkboxes getting deselected
```

Comment:
Can we improve this entry to reflect it on the setting page and add some clarification - this is an external type of communication.

Reply:
Thanks, @kalessil - I have updated it to more clear and elaborate message here : https://github.com/Automattic/woocommerce-payments/pull/5171/commits/ce84518479ebbf901852ce5e47d770457f58e35b

Reply:
Yay, that's much better! Thanks.